subprojects/core-api/src/main/java/org/gradle/StartParameter.java

```diff
@@ -353,7 +356,7 @@ public void setTaskRequests(Iterable<? extends TaskExecutionRequest> taskParamet
     /**
      * Sets the tasks to exclude from this build.
      *
-     * @param excludedTaskNames The task names. Can be null.
+     * @param excludedTaskNames The task names.
      */
     public void setExcludedTaskNames(Iterable<String> excludedTaskNames) {
```

Comment:
Shouldn't this be annotated with `@Nullable`?

Reply:
Sorry, you are right. I misread the diff thinking the add was a subtract. My mistake.